helm/kong-app/values.yaml

```diff
@@ -495,7 +495,7 @@ ingressController:
     # The controller disables TLS verification by default because Kong
     # generates self-signed certificates by default. Set this to false once you
     # have installed CA-signed certificates.
-    kong_admin_tls_skip_verify: "true"
+    kong_admin_tls_skip_verify: true
```

Comment:
Why is this changed? By upstream? Maybe we should still add one line to changelog?

Reply:
Yes, from upstream. Forgot to update the values in previous sync. This is related to a change they reverted themselves in upstream already (That was related to requiring all `env` values to be of type string). So I would not include this in Changelog.

Reply:
Introduced by https://github.com/Kong/charts/pull/728
Reverted in https://github.com/Kong/charts/pull/733/files